amp/repository/aim/view/teamAuditList.jsp

```diff
@@ -374,3 +390,7 @@ function showUser(email){
 </table>
 
 </digi:form>
+<digi:form action="/compareActivityVersions.do" method="post" type="aimCompareActivityVersionsForm">

```

Comment:
although this will get more changes in the next ticket it will look the same as the one we have in audit logger. here we can use a jspf to define the form and include it in both files,